manual/configuration.md

```diff
@@ -302,7 +302,7 @@ In `.rubocop.yml` and any other configuration file beginning with `.rubocop`,
 files, and directories are specified relative to the directory where the
 configuration file is. In configuration files that don't begin with `.rubocop`,
 e.g. `our_company_defaults.yml`, paths are relative to the directory where
-`rubocop` is run.
+RuboCop is run.
```

Comment:
I think it means a command name, not a product name, because it is enclosed in back quotes.

Reply:
I misunderstood.
I fixed it.